src/cr/cube/matrix/measure.py

```diff
@@ -38,12 +44,32 @@ def column_comparable_counts(self):
 
     @lazyproperty
     def column_index(self):
-        """_ColumnIndex measure object for this cube-result."""
+        """_ColumnIndex measure object for this cube-result.
+
+        If smoothing is defined in the dimension transform a _ColumnIndexSmoothed object
+        will be returned.
+        """
+        column_dimension = self._dimensions[-1]
+        smoothing_spec = column_dimension.smoothing_spec(MEASURE.COLUMN_INDEX)
+        if smoothing_spec:
+            return _ColumnIndexSmoothed(
+                self._dimensions, self, self._cube_measures, smoothing_spec
+            )
         return _ColumnIndex(self._dimensions, self, self._cube_measures)
 
     @lazyproperty
     def column_proportions(self):
-        """_ColumnProportions measure object for this cube-result."""
+        """_ColumnProportions measure object for this cube-result.
```

Comment:
So one of the things we need to think about in the design is how we could provide _both_ smoothed and unsmoothed values. The idea is that a chart would have the smoothed values as the line and the unsmoothed values as "points", kind of bouncing around on either side of that line.

For this use case, it won't do to have smoothing be unconditionally automatic. We need to be able to get both.

We'll need to noodle how they ask for this too, because that's going to matter. Either it can be by asking for both measures, like `"measures": ["col_percent", "smoothed_col_percent"]` or it could be by making the smoothed_col_percent item an insertion, like a new row. I don't think this second one is the right answer, but I'm not 100% sure the first one is right either.

Anyway, something for us to noodle on.

In any case, I'm thinking that `"smoothed_col_percent"` kind of makes sense, in the way that `significant_columns` does, where you ask for it in one place and then specify what the parameters are in another. Like we have thresholds for significance that are separate from `significant_columns`. And maybe we can decide that all smoothing will be done the same way, regardless of measure, so there can just be an optional `"smoothing":` field in transforms and we don't need separate ones for each measure (because that spec no longer needs to specify which measures to smooth). Then we could have a default smoothing method and parameters, like (one-sided, window=3) so we don't even have to care whether the user specified or not.

The only thing left is how you specify which axis to smooth along. I'm thinking it's always the one with a CAT_DATE type, or maybe defaulting to smoothing-across-columns. Anyway, a bit more for us to noodle. Maybe that can go in the smoothing spec as well, defaulting to the CAT_DATE dimension or columns.